### ISSUE TITLE
doesn't recognise focal fossa 20 iso

### ISSUE THREAD

**Issue by pranav517517** [mbusb/multibootusb#511]
my os is focal fossa 20
i tried to installed it in a pendrive as well as harddrive through multiboot usb but doesn't get installed properly

my config are
os-focal fossa
file partition of pendrive and harddrive is ntfs, fat32 and yes i tried with all of the file system
iso file-ubuntu focal fossa 20

**Comment by carriba** [comment]
Not sure to understand the above, however to complement: 

I'm running into issues that I cannot include UbuntuMATE Focal Fossa 20.04 LTS (ISO available under https://ubuntu-mate.org/download/amd64/focal) on my FAT32 formatted USB drive with the latest version of MultiBootUSB 9.2.0 installed on a Windows 10 (64-bit) platform. The USB drive has enough disk space available!

Though, the inclusion on my USB drive works fine with UbuntuMATE Bionic Beaver 18.04.04 LTS (ISO available under https://ubuntu-mate.org/download/amd64/bionic). 

Available to provide further glues or additional information to troubleshoot further...

**Comment by sneakyx** [comment]
Yes, it is the same, but it worked with Multisystem - so I changed now to that software.
I had the same issue

**Comment by aelfinn** [comment]
Same here. Tested with the official ISOs for Kubuntu 20.04 and Ubuntu Budgie 20.04.

**Comment by robotshateme** [comment]
Same.  Project looks abandoned which is a shame.

**Comment by carriba** [comment]
Project is active from what I see, and have been able to debug above observations today.

**Comment by aelfinn** [comment]
@carriba – What have you been able to find out?

**Comment by josuesoto1** [comment]
Any luck installing Ubuntu 20.04 with multi-boot and persistence option?
any other application that has multi boot and persistence that works within Ubuntu?

**Comment by Philippe734** [comment]
The Linux version of Multibootusb is not support 20.04 iso. The author mbusb didn't reply since one year (jun 2019). I think too the project is look abandoned. Who is and where is the project leader ? How to help to add support to iso 20.04 for the Linux version ?

**Comment by Philippe734** [comment]
> Any luck installing Ubuntu 20.04 with multi-boot and persistence option?
> any other application that has multi boot and persistence that works within Ubuntu?

A new tool works fine and it's support 20.04: [here is ventoy](https://github.com/ventoy/Ventoy)
I will make a GUI for ventoy.

**Comment by aelfinn** [comment]
> A new tool works fine and it's support 20.04: [here is ventoy](https://github.com/ventoy/Ventoy)
> I will make a GUI for ventoy.

Have you actually made any progress with the GUI?

**Comment by aelfinn** [comment]
One interesting data point: The .iso of EnsoOS, which AFAIK is based on Xubuntu 20.04, works with MultibootUSB without a problem. Any idea why that would be?
